Slim/Player/StreamingController.pm

```diff
@@ -916,11 +916,23 @@ sub _RetryOrNext {		# -> Idle; IF [shouldretry && canretry] THEN continue
 		&& $song->isRemote()
 		&& $elapsed > 10)				# have we managed to play at least 10s?
 	{
-		if (!$song->duration() && $song->isLive()) {	# unknown duration => assume radio
+		if (!$song->duration() || $song->isLive()) {	# unknown duration => assume radio
```

Comment:
So we are going there as soon as a track as no duration, which might not be just remote. Would have to check but when playing local track from directory browsing, we might not have scanned it yet. Might not be a big deal, but that's a key change of logic

Reply:
Not mateirial to the change. Reverting.